components/Chips/src/TypographyThemer/MDCChipViewTypographyThemer.m

```diff
@@ -22,7 +22,7 @@ @implementation MDCChipViewTypographyThemer
 + (void)applyTypographyScheme:(nonnull id<MDCTypographyScheming>)typographyScheme
                    toChipView:(nonnull MDCChipView *)chipView {
   UIFont *titleFont = typographyScheme.body2;
-  if (typographyScheme.mdc_adjustsFontForContentSizeCategory) {
+  if (typographyScheme.useCurrentContentSizeCategoryWhenApplied) {
```

Comment:
useCurrentContentSizeCategoryWhenApplied is an optional property on the protocol. Please check for its existence prior to calling it.

Reply:
Good catch. Why is this property an optional one on `MDCTypographyScheming`? I think we need to make `MDCTypographyScheming` conform to <NSObject> in this case, otherwise we are not able to check if it responds to this optional property.

Reply:
I sent a PR https://github.com/material-components/material-components-ios/pull/7485 for it.

Reply:
It's optional so that we can gradually migrate internal clients to the behavior and not require this to be a breaking change for the next release.